package.json

```diff
@@ -102,6 +104,11 @@
     "stylelint-scss": "^2.0.0",
     "stylelint-suitcss": "^1.0.0",
     "terra-toolkit": "^2.2.0",
+    "wdio-mocha-framework": "^0.5.11",
+    "wdio-static-server-service": "^1.0.0",
```

Comment:
I'm adding tests then putting out a PR for WebPackDevService so we can drop the wdio-static-server-service & continue to use the webpack-dev-server as expected. Then we can drop the `packages/terra-site/dist/` from .gitIgnore

Reply:
@emilyrohrbough What's the status on this?

Reply:
We’re close to pushing a release of toolkit which will clean up this or a bunch